lib/pronto/git/repository.rb

```diff
@@ -13,6 +13,18 @@ def diff(commit, options = nil)
                             [head_commit_sha, @repo.index.diff(options)]
                           when :staged
                             [head_commit_sha, head.diff(@repo.index, options)]
+                          when :workdir
+                            [
+                              head_commit_sha,
+                              @repo.diff_workdir(
+                                head,
+                                {
+                                  include_untracked: true,
+                                  include_untracked_content: true,
+                                  recurse_untracked_dirs: true,
```

Comment:
I think it's complaining about this extra comma.

Reply:
Ugh, you're right. Fixed and pushed.